shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/LedgerState.hs

```diff
@@ -88,6 +98,7 @@ module Shelley.Spec.Ledger.LedgerState
     -- *
     TransUTxOState,
     TransLedgerState,
+    UsesPP,
```

Comment:
Shouldn't export this, at least not with this name

Reply:
No longer exported, and a comment applied which explains where this is used and why.